.github/workflows/build.yml

```diff
@@ -0,0 +1,26 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup JDK
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: '8'
```

Comment:
8 ? Not 11 ?

Reply:
Why would you want to drop Java 8 support here? Doesn’t Kotlin support Java 8?

Reply:
My bad. Security supported until 31 Dec 2030.